pom.xml

```diff
@@ -64,6 +64,12 @@
             <artifactId>workflow-step-api</artifactId>
             <version>${workflow.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>ws-cleanup</artifactId>
+            <version>0.30-SNAPSHOT</version>
+            <optional>true</optional>
```

Comment:
I remember that we have discussed to make it `optional`, but maybe I've misunderstood.